src/Worlds.js

```diff
@@ -26,58 +36,95 @@ import { ReactComponent as SpaceContrast } from './svg/Space-contrast.svg';
 import { ReactComponent as SpaceThumbnail } from './svg/SpaceThumbnail.svg';
 import { ReactComponent as SpaceThumbnailGray } from './svg/SpaceThumbnail-gray.svg';
 import { ReactComponent as SpaceThumbnailContrast } from './svg/SpaceThumbnail-contrast.svg';
-
-// Characters
-import { ReactComponent as RabbitIcon } from './svg/Rabbit.svg';
-import { ReactComponent as RobotIcon } from './svg/Robot.svg';
-import { ReactComponent as SpaceShipIcon } from './svg/SpaceShip.svg';
+import { ReactComponent as SpaceShip } from './svg/SpaceShip.svg';
+import { ReactComponent as SpaceShipGray } from './svg/SpaceShip-gray.svg';
+import { ReactComponent as SpaceShipContrast } from './svg/SpaceShip-contrast.svg';
 
 export type WorldProperties = {
-    background: any,
-    backgroundGray: any,
-    backgroundContrast: any,
-    thumbnail: any,
-    thumbnailGray: any,
-    thumbnailContrast: any,
-    character: any
+    background: ?React.ComponentType<{}>,
+    backgroundGray: ?React.ComponentType<{}>,
+    backgroundContrast: ?React.ComponentType<{}>,
+    thumbnail: React.ComponentType<{}>,
+    thumbnailGray: React.ComponentType<{}>,
+    thumbnailContrast: React.ComponentType<{}>,
+    character: React.ComponentType<{}>
 };
 
 const worlds = {
     'DeepOcean': {
         background: DeepOcean,
         backgroundGray: DeepOceanGray,
         backgroundContrast: DeepOceanContrast,
+        backgroundInfo: [
```

Comment:
I understand that a 2-dimensional array is a simple way of expressing this, but I think there are other ways to encode this that would require less `null` values, and that would be more flexible if we decide to change the scene size again.

For example:

```
{
  1: { 8: "DeepOcean.fishGroup", 9: "DeepOcean.fishGroup" },
  2: { 8: "DeepOcean.fishGroup", 9: "DeepOcean.fishGroup" },
  3: { 1: "DeepOcean.babyJellyfish", 2: "DeepOcean.babyJellyfish", 6: "DeepOcean.shark", 7: "DeepOcean.shark", 10: "DeepOcean.shark", 11: "DeepOcean.coral" }
  // etc.
}
```

You'd need a slightly different check to see if it's set, something like:

```
let cellContent = backgroundInfo[row] && backgroundInfo[row][col];
if (cellContent) { 
  // Do stuff...
}
```

But you can omit all empty cells and even whole empty rows with that kind of approach, and still  check for content in the cell easily.